### ISSUE TITLE
[Accessibility][Keyboard -New Dialog]: Mandatory sign is not defined for name in New dialog.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2073]
**User Experience** 
User with vision disability will not get any information about the required field. He might be confused with this.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to 'New Dialog' button available in the filter dialog pane and activate it..
4. Type the name of the dialog which is already created ex- Actions.
5. Observer the issue.

**Actual:** 
The name is mandatory fields but no visual or audio information is conveyed to the user about it.

**Expected:**
Their should be astreak symbol for the mandatory field as well as the screen reader should inform about it, so that the user will aware about that.

**MAS Impacted:** [MAS3.3.4](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={5f773cee-c1c7-4292-b330-cb2f4aad00c4})

**Attachment:**
[MAS3.3.4_Mandatory sign is not defined for name in New dialog.zip](https://github.com/microsoft/BotFramework-Composer/files/4249012/MAS3.3.4_Mandatory.sign.is.not.defined.for.name.in.New.dialog.zip)
![MAS3 3 4_Mandatory sign is not defined for name in New dialog](https://user-images.githubusercontent.com/60913131/75231769-0fef8b00-57dc-11ea-8b56-b920ea28e025.png)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.
**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
